### DISCUSSION TITLE
How to remove index.php

### DISCUSSION THREAD

**Discussion by cngirwa** [LiveHelperChat/livehelperchat#1888] [Q&A] [answered]
Hello!

I'm on v.4.06 and followed the instructions on how to remove index.php posted [here](https://doc.livehelperchat.com/docs/development/remove-index-php).

When changing the value from 
> 'force_virtual_host' => false 

to
> 'force_virtual_host' => true

When I try to sign in, the system navigates to > https://livechat.mydomain.com/site_admin/user/login crashes, and displays an error message below

> Not Found
> The requested URL was not found on this server.

The instructions say after changing the value to "true", I should clear the cache from the back office. But I'm unable to do so since I can't access the system. Please assist

**Comment by remdex** [comment]
https://doc.livehelperchat.com/docs/development/remove-index-php

**Reply by cngirwa** [reply]
I tried to follow the suggested URL with no success... the error keep saying 

> Not Found
The requested URL was not found on this server.

**Comment by cngirwa** [comment, marked as answer]
Found an answer to this question:

1. Make sure you are signed in and navigate to Settings
2. Change setting/settings.ini.php `'force_virtual_host' => true`
3. Clean the cache in back office (about 2 or 3 times) until you see the error page
4. Edit .htaccess as suggested [here](https://doc.livehelperchat.com/docs/development/remove-index-php/)
5. Enable rewrite mod as suggested [here](https://stackoverflow.com/questions/869092/how-to-enable-mod-rewrite-for-apache-2-2)
6. Enable headers using` a2enmod headers `

Try to access your livechat without index.php

**Reply by GuilhermeDeLucca** [reply]
to use rewrite you have to enable this in apache, you have to see how your vhost structure is, I believe the easiest way to do this is

<Directory /mnt/www/html/web_lhc>
                        AllowOverride All
                </Directory>
                
                
           
Oh also don't forget to add {ico} here RewriteRule !\.(gif|ico|.....
